content/guides/anomalies.md

```diff
@@ -73,6 +73,14 @@ No. Anomaly detection is designed to assist with visualizing and monitoring metr
 
 Also, anomaly detection requires historical data to make good predictions. If you have only been collecting a metric for a few hours or a few days, anomaly detection probably won't be very useful.
 
+### For the seasonal algorithms (_robust_, _agile_, and _adaptive_), how much history is analyzed? What types of seasons are included?
+
+All of the seasonal algorithms may use up to a couple of months of historical data when calculating a metric's expected normal range of behavior. By using a significant amount of past data, the algorithms are able to avoid giving too much weight to abnormal behavior that might have occurred in the recent past.
+
+_Robust_ and _agile_ assume that the metric has weekly seasonality.
```

Comment:
i guess that makes it a bit short. maybe add the graph about data to the algorithms section? Like "We currently offer four different anomaly detection algorithms. All of the seasonal... "